modules/oauth-customizing-the-oauth-server-URL.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * authentication/configuring-internal-oauth.adoc
+
+[id="customizing-the-oauth-server-url_{context}"]
+= Customizing the internal OAuth server URL
+
+You can customize the internal OAuth server URL by setting the custom host name and TLS certificate in the `spec.componentRoutes` field of the cluster `Ingress` configuration.
```

Comment:
I'm not sure if `host name` shouldn't be spelled together (`hostname`).

Reply:
Thanks for mentioning. But yeah this is actually not a clear-cut answer right now. Here's more info than you actually want to know :wink: :

Our style guide that we follow used to recommend "host name" two words, so that's what we've usually tried to follow. But a recent update of our style guide changed it to recommending "hostname" one word. We have a discussion in progress to determine which to follow, but for now, most of our docs still follow the two word option. So I'd keep it two words for now, for consistency. :)

Reply:
Ok, thanks for the context - I'm fine either way. Just more used to `hostname`. But don't want to start a vim vs emacs type of war :)

Reply:
Following up on this, we just came to a consensus to user "hostname" one word (and I updated all instances in our repo), so :partying_face:

Reply:
Woohoo! :tada: